scripts/src/cowidev/vax/incremental/myanmar.py

```diff
@@ -0,0 +1,122 @@
+import re
+
+from bs4 import BeautifulSoup, element
+import pandas as pd
+
+from cowidev.utils.web import get_soup
+from cowidev.utils.clean import clean_count, clean_date
+from cowidev.vax.utils.incremental import increment, enrich_data
+
+
+class Myanmar:
+    location = "Myanmar"
+    _base_url = "https://mohs.gov.mm"
+    _url_subdirectory = "/main/content/new/list?pagesize=9&pagenumber="
+    _num_max_pages = 3
+    regex = {
+        "title": r"ကိုဗစ်-19 ရောဂါ ကာကွယ်ဆေး ထိုးနှံပြီးစီးမှု",
+        "date": r"(\d{1,2}\-\d{1,2}\-20\d{1,2})",
```

Comment:
@lucasrodes 
```python
r"(\d{1,2}\-\d{1,2}\-20\d{2}"
```
I would be grateful if you fix it ( too much of a hassle to be a PR)

Reply:
Would you mind elaborating on what is to be fixed? Thanks!